js_machine_front/src/components/error-boundry/error-boundry.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';

```

Comment:
typo, rename component to error-boundary

Reply:
Fixed